Sources/AppKit/NSCollectionView+Changeset.swift

```diff
@@ -0,0 +1,24 @@
+import AppKit
+import ReactiveSwift
+
+extension Reactive where Base: NSCollectionView {
+	@available(macOS 10.11, *)
+	public func apply(_ changeset: Changeset, toSection section: Int) {
```

Comment:
These apply methods feel a little weird to me on `Reactive` since there's nothing reactive about them.